.github/ISSUE_TEMPLATE/bug_report_template.yaml

```
@@ -0,0 +1,158 @@
+name: Bug Report
+description: Create a report to help us reproduce and correct the bug
+labels: ['bug']
+title: '[BUG]'
+
+body:
+  - type: markdown
+    attributes:
+      value: | 
```

```suggestion
      value: |
```

Let's remove trailing spaces.